plugins/system/kernel_same_page_merging

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
```

Comment:
I think, you are not using any bash-specific features. Maybe use `/bin/sh` instead?